deployment/scripts/deployment.sh

```diff
@@ -30,8 +30,10 @@ sudo chcon -t httpd_sys_rw_content_t $TARGET/../env/$PROJECT/ $TARGET/generated/
 
 echo "Composer auth.json"
 cd $TARGET
-mv -f $TARGET/deployment/auth.json /var/www/.composer/auth.json
+/bin/cp -f $TARGET/deployment/auth.json /var/www/.composer/auth.json
+/bin/cp -f $TARGET/deployment/auth.json $TARGET/var/composer_home/auth.json
```

Comment:
This is going to fail the first deployment because that folder does not exist. You will need to either create the folder, or copy auth.json after 'composer install'